docs-new/docs/graphql/core/databases/ms-sql-server/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: Hasura MS SQL Server database support
+keywords:
+  - hasura
+  - docs
+  - databases
+  - ms sql
+  - sql server
+---
+
+# MS SQL Server {#database-ms-sql-server}
+
+## Introduction
+
+Hasura allows connecting to a SQL Server database and build a GraphQL
+API based on the database schema.
+
+:::note Supported versions:
+
+1.  Hasura GraphQL engine `v2.0.0-alpha.2` onwards
+2.  SQL Server 2016 and upwards
+
+:::
+
+## Get Started
+
+To try Hasura with SQL Server, you'll need your own new or existing SQL
+Server database.
+
+Here are 2 ways you can get started with Hasura and SQL Server:
+
+1.  [Hasura Cloud](/graphql/core/databases/ms-sql-server/getting-started/cloud.mdx): You'll need to be
+    able to access your SQL Server database from Hasura Cloud.
+2.  [Docker](/graphql/core/databases/ms-sql-server/getting-started/docker): Run Hasura with Docker and
+    then connect your SQL Server database to Hasura.
+
+## Supported features
+
+Hasura currently supports queries, subscriptions, inserts, deletes,
+relationships and permissions on MS SQL Server.
+
+Next up on our roadmap for Hasura + SQL Server:
+
+-   Support for stored procedures & functions
+    ([#7073](https://github.com/hasura/graphql-engine/issues/7073))
+-   Mutations: Run updates, stored procedures and transactions securely
+    on SQL Server over a GraphQL API
+    ([#7074](https://github.com/hasura/graphql-engine/issues/7074))
+-   Event triggers: Trigger HTTP webhooks with atomic capture and
+    atleast once guarantee whenever data changes inside the database
+    ([#7075](https://github.com/hasura/graphql-engine/issues/7075))
+-   Remote Joins: Join models in SQL Server to models from other API
+    services (GraphQL or REST)
+    ([#7076](https://github.com/hasura/graphql-engine/issues/7076))
+
+## Keep up to date
+
+If you'd like to stay informed about the status of SQL Server support,
+subscribe to our newsletter and join our discord!
+
+-   [https://hasura.io/newsletter/](https://hasura.io/newsletter/)
+-   [https://discord.com/invite/hasura](https://discord.com/invite/hasura)
+
+:::note Additional Resources
+
+This Hands-on Demo walks you through Getting Started with Hasura on SQL
+Server & common use cases. - [View Recording
+here](https://hasura.io/events/webinar/hasura-sql-server/?pg=docs&plcmt=body&cta=view-recording&tech=).
+
+:::
+
+## Know more
+
+- [Getting Started](docs-new/docs/graphql/core/databases/ms-sql-server/getting-started/index.mdx)
+- [Schema](docs-new/docs/graphql/core/databases/ms-sql-server/schema/index.mdx)
+- [Queries](docs-new/docs/graphql/core/databases/ms-sql-server/queries/index.mdx)
+- [Mutations](docs-new/docs/graphql/core/databases/ms-sql-server/mutations/index.mdx)
+- [Subscriptions](docs-new/docs/graphql/core/databases/ms-sql-server/subscriptions/index.mdx)
+
+/*
+.. TODO: DB-COMPATIBILITY
+
+  .. toctree::
+    :maxdepth: 1
+    :titlesonly:
+
+    Supported MS SQL Server types <mssql-types>
+*/
```

Comment:
This is a JavaScript comment and will show up in UI. Please try the markdown commenting.

```
<!--
.. TODO: DB-COMPATIBILITY

  .. toctree::
    :maxdepth: 1
    :titlesonly:

    Supported MS SQL Server types <mssql-types>
-->
```